.github/workflows/nextjs_bundle_analysis.yml

```diff
@@ -0,0 +1,100 @@
+name: 'Next.js Bundle Analysis'
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  # In case of multiple commits of same PR, the previous run of the Action will be deleted
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read # for checkout repository
+  actions: read # for fetching base branch bundle stats
+  pull-requests: write # for comments
+
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3.7.0
```

Comment:
What do you think about aligning the version of `setup-node` action? The `ci` file uses a different one (`actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c`)